kolibri/content/models.py

```diff
@@ -144,102 +136,51 @@ def get_url(self):
             return None
 
 
-class License(models.Model):
+@python_2_unicode_compatible
+class License(ContentDatabaseModel):
     """
     Normalize the license of ContentNode model
     """
     license_name = models.CharField(max_length=50)
 
     objects = ContentQuerySet.as_manager()
 
-    class Admin:
-        pass
-
     def __str__(self):
         return self.license_name
 
 
-class PrerequisiteContentRelationship(models.Model):
+@python_2_unicode_compatible
+class ChannelMetadataAbstractBase(models.Model):
     """
-    Predefine the prerequisite relationship between two ContentNode objects.
+    Holds metadata about all existing content databases that exist locally.
     """
-    target_node = models.ForeignKey(ContentNode, related_name='%(app_label)s_%(class)s_target_node')
-    prerequisite = models.ForeignKey(ContentNode, related_name='%(app_label)s_%(class)s_prerequisite')
-
-    objects = ContentQuerySet.as_manager()
+    id = UUIDField(primary_key=True)
+    name = models.CharField(max_length=200)
+    description = models.CharField(max_length=400, blank=True)
+    author = models.CharField(max_length=400, blank=True)
+    version = models.IntegerField(default=0)
+    thumbnail = models.TextField(blank=True)
+    root_pk = UUIDField()
 
     class Meta:
-        unique_together = ['target_node', 'prerequisite']
+        abstract = True
 
-    class Admin:
-        pass
-
-    def clean(self, *args, **kwargs):
-        # self reference exception
-        if self.target_node == self.prerequisite:
-            raise IntegrityError('Cannot self reference as prerequisite.')
-        # immediate cyclic exception
-        elif PrerequisiteContentRelationship.objects.using(self._state.db)\
-                .filter(target_node=self.prerequisite, prerequisite=self.target_node):
-            raise IntegrityError(
-                'Note: Prerequisite relationship is directional! %s and %s cannot be prerequisite of each other!'
-                % (self.target_node, self.prerequisite))
```

Comment:
Yeah, I was assuming they're already being enforced on the content curation server -- relationships aren't being created locally in the Kolibri app, so no need to enforce here.